### PR TITLE
ticket 9 completed with testing

### PR DESCRIPTION
**Pull request by EddTheDuck**

### CODE REVIEW

**Review by karlaferry — CHANGES_REQUESTED** [review]
Please add one more error handling test, what if the article_id is valid but doesn't exist?

File: __tests__/app.test.js [hunk 1]
```diff
@@ -218,3 +218,47 @@ describe("GET /api/articles", () => {
...
+          expect(typeof comment.votes).toBe("number");
+          expect(typeof comment.created_at).toBe("string");
+          expect(typeof comment.author).toBe("string");
+          expect(typeof comment.body).toBe("string");
+        });
+      });
+  });
+  test("responds with 404: valid ID but doesn't exist", () => {
+    return request(app)
+      .get("/api/articles/2828")
```
Comment (karlaferry) [inline __tests__/app.test.js:258]: Could you do one for `/api/articles/2828/comments`?

**Review by karlaferry — CHANGES_REQUESTED** [review]
Please use the correct path in the test.

File: __tests__/app.test.js [hunk 2]
```diff
@@ -218,3 +218,40 @@ describe("GET /api/articles", () => {
...
+  test("returns status 400 when given invalid id ", () => {
+    return request(app)
+      .get("/api/articles/test/comments")
+      .expect(400)
+      .then(({ body }) => {
+        expect(body.msg).toBe("Bad Request!");
+      });
+  });
+
+  test("returns status 200 and an empty array when an article has no comments ", () => {
```
Comment (Rosem10) [inline __tests__/app.test.js:231]: One more test please for a request made to an article that doesn't exist but is the right type (e.g 99999) - you shouldn't have to change any of your code but it needs to be covered by a test!

**Review by Rosem10 — CHANGES_REQUESTED** [review]
Good to go once you've written that test. Don't think you'll have any bother, but let me know if you run into trouble!

File: __tests__/app.test.js (hunk 2 above)
Comment (EddTheDuck) [reply __tests__/app.test.js:231]: I thought I had already tested for this?
Screenshot 2022-08-05 at 14 50 15

**Review by karlaferry — APPROVED** [review]
All good!

**Pull request merged**